src/routes/journey/__tests__/journeyEditFlow.tsx

```diff
@@ -36,6 +39,7 @@ beforeEach(() => {
   (updateChallengeNote as jest.Mock).mockReturnValue(updateChallengeNoteResult);
   (editComment as jest.Mock).mockReturnValue(editCommentResult);
   (getJourney as jest.Mock).mockReturnValue(getJourneyResult);
+  (trackScreenChange as jest.Mock).mockReturnValue(trackScreenChangeResult);
```

Comment:
A similar thing, had to add this in order to fix the test. Not sure why it wasn't needed before.